com.modrinth.ModrinthApp.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.modrinth.ModrinthApp</id>
+  <launchable type="desktop-id">com.modrinth.ModrinthApp.desktop</launchable>
+  <name>Modrinth App</name>
+  <developer_name>Modrinth</developer_name>
+  <summary>Modrinth's game launcher</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <url type="homepage">https://modrinth.com</url>
+  <url type="bugtracker">https://github.com/modrinth/theseus/issues</url>
+  <url type="vcs-browser">https://github.com/modrinth/theseus</url>
+  <url type="contribute">https://github.com/modrinth/theseus</url>
+  <description>
+    <p>A unique, open source launcher that allows you to play your favorite mods, and keep them up to date, all in one neat little package</p>
+  </description>
+	<!-- this doesn't work...why?
```

Comment:
webp isn't supported. thats why.
and screenshot are mandatory.

Reply:
i'll have to work upstream then. they currently don't offer any other formats for screenshots to my knowledge